extensions/mssql/src/services/sqlTasksService.ts

```diff
@@ -65,12 +66,63 @@ type ActiveTaskInfo = {
 };
 type ProgressCallback = (value: { message?: string; increment?: number }) => void;
 
+/**
+ * Configuration for a custom task completion handler that shows a notification with an action button
+ */
+export interface TaskCompletionHandler {
+    /**
+     * The name of the task to handle (must match taskInfo.name from SQL Tools Service)
```

Comment:
are these taskNames localized at all when coming to SQL Tools Service?  I don't love matching on a human-friendly string (rather than a task type ID that's less likely to change and won't get localized).